docs/azure-openshift.md

```diff
@@ -37,10 +37,15 @@ openshift-console          downloads           downloads-openshift-console.apps.
 
 6. Get the list of dns zones w.r.t your resource group to find the one which corresponds to the previously found route’s domain:
 ```bash
-$ az network dns zone list --resource-group "${RESOURCE_GROUP}"
+$ ZONE_NAME=$(az network dns zone list -g "${RESOURCE_GROUP}" -o tsv --query '[0].name')
```

Comment:
Is there a guarantee that there is one and only one dns zone?

Reply:
No, there can be many zones. The previous command was a better suit here. I just wanted to set a `$ZONE_NAME` environment variable to use it down the doc. Let me update the command to make sure only one zone is selected:
```
$ az network dns zone list -g "${RESOURCE_GROUP}" -o tsv --query '[].name'
example.com
anotherdomain.net
test-azure.qe.azure.devcluster.openshift.com

$ ZONE_NAME="test-azure.qe.azure.devcluster.openshift.com"
```